draft-ietf-openpgp-pqc.md

```diff
@@ -1542,6 +1566,18 @@ second-preimage resistance.
 In the case of SLH-DSA the internal hash algorithm varies based on the
 algorithm and parameter ID, see {{slhdsa-sig-data-digest}}.
 
+## Symmetric Algorithms for SEIPD Packets
+
+This specification mandates support for `AES-256` for two reasons.
+First, `AES-KeyWrap` with `AES-256` is already part of the composite KEM construction.
+Second, some of the PQ(/T) algorithms target the security level of `AES-256`.
+
+For the same reasons, this specification further recommends the use of `AES-256`
+if it is supported by all recipients, regardless of what the implementation
+would otherwise choose based on the recipients' preferences.
+This recommendation gives a clear and easy rule by which to use `AES-256`.
```

Comment:
Do you mean the ones in section "Symmetric Algorithms for SEIPD Packets"?

If yes, I would make it plural ("clear and easy rules") and add a link to that section.